### ISSUE TITLE
Circulating blood brown or black

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11163]
**Preferred term label:**
Circulating blood brown or black

**Synonyms**
Chocolate color

**Definition (free text, please give PubMed ID)**
https://uts.nlm.nih.gov/uts/umls/concept/C2237992 but it would be better to specify that the blood is circulating not vomited.

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Methemoglobinemia

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
There is a term that refers to the actual measurement [Methemoglobinemia HP:0012119 ](https://hpo.jax.org/browse/term/HP:0012119). Just to confirm this term is meant to be the physical appearance, which might be useful if the physician does not know what Methemoglobinemia is?

**Comment by MickeySegal** [comment]
Correct.  We also use  https://hpo.jax.org/browse/term/HP:0012119 in these diseases, but the requested finding is meant to be an observation by the clinician or by history from the patient that could result in more rapid diagnosis.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001452
- label: Chocolate brown blood
- parents: [Abnormality of blood and blood-forming tissues (HP:0001871)](https://hpo.jax.org/app/browse/term/HP:0001871)
- pmid: PMID:30726002
- def: Characteristic chocolate-brown color of circulating blood with an elevated methemoglobin content.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11163
- synonyms: Circulating blood brown or black(NONE)
